src/reanimated2/animation/util.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-shadow */
```

Comment:
Why do we disable this rule?

Reply:
Since it requires almost rewriting this file, which is very important one. I would prefer to do it in some other PR.

Reply:
@Latropos Let's add a TODO here and a respective task to our Trello then